.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "files.associations": {
+        "cstdlib": "cpp"
+    }
+}
```

Comment:
New line missing

Reply:
This file is not required and it is removed it in fb2a3eb